### PR TITLE
New version 2.0

### PR DESCRIPTION
**Pull request by BrOrlandi**

### CODE REVIEW

File: src/html/donateButtons.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      <input type="hidden" name="hosted_button_id" value="AU6WSD3YHLGHY">
+      <input type="hidden" name="charset" value="utf-8">
+      <input type="image" src="https://www.paypalobjects.com/pt_BR/i/btn/btn_donate_LG.gif" border="0" name="submit" alt="PayPal - A maneira fácil e segura de enviar pagamentos online!">
+      <input type="hidden" name="custom" value="">
+      <img alt="" border="0" src="https://www.paypalobjects.com/pt_BR/i/scr/pixel.gif" width="1" height="1">
+  </form>
+</div>
+</div>
+<script>
+  var isFromBrazil = () => {
```
Comment (luanorlandi) [inline src/html/donateButtons.html]: These var can be const

Comment (BrOrlandi) [reply src/html/donateButtons.html]: This file is not being parsed by babel, so I'm using es5 for better compatibility.

Comment (luanorlandi) [reply src/html/donateButtons.html]: Yes, but on [lines 49 and 50](https://github.com/KasselLabs/StarWarsIntroCreator/pull/1/files/06354bcfad3ef837c8587f971d54efbb36c9a69c#diff-69cb7b2c66f612269ee5191109d61a19R49) in this same file we are using const, so they could be var to keep up to ES5

Comment (luanorlandi) [reply src/html/donateButtons.html]: Fixed

**Pull request merged**